### DISCUSSION TITLE
Is it possible to support mono c# to build for x86 architecture in the near future for android/ios?

### DISCUSSION THREAD

**Discussion by Famouzy** [godotengine/godot-proposals#8855] [Scripting]
### Describe the project you are working on

Android game project

### Describe the problem or limitation you are having in your project

Cant build c# project on x86 acrchitectures for android

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Support for x86 architecture in yearot mono is very much to the point for phones with this very CPU architecture

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

It is very important to assemble the project for 2 types of architectures

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.
